x-pack/metricbeat/module/aws/utils.go

```diff
@@ -29,11 +26,14 @@ func GetStartTimeEndTime(period time.Duration, latency time.Duration) (time.Time
 		endTime = endTime.Add(latency * -1)
 	}
 
-	// Set startTime double the period plus one second earlier than the endTime in order to
-	// make sure GetMetricDataRequest gets the latest data point for each metric. The plus
-	// one second is to make sure the startTime of the next collect period is one second later
-	// than the endTime of last collection period. This is to avoid collecting duplicate data.
-	return endTime.Add(period*-2 + time.Second), endTime
+	// Set startTime to be one period earlier than the endTime. If metrics are
+	// not being collected, use latency config parameter to offset the startTime
```

Comment:
should latency config usage to overcome not collected metrics be mentioned also in docs? (if it's not already)